src/mono/wasm/debugger/DebuggerTestSuite/DebuggerTestSuite.csproj

```diff
@@ -14,6 +14,9 @@
     <!-- Set RunSettingsFilePath property which is read by VSTest. -->
     <RunSettingsFilePath Condition="'$(ContinuousIntegrationBuild)' == 'true'">$(OutputPath).runsettings</RunSettingsFilePath>
     <RunSettingsFilePath Condition="'$(ContinuousIntegrationBuild)' != 'true'">$(ArtifactsObjDir)vscode\.runsettings</RunSettingsFilePath>
+
+    <!-- some test require full ICU data, force full ICU -->
+    <IcuFileName>icudt.dat</IcuFileName>
```

Comment:
Instead of this, use a property like `$(WasmIncludeFullIcuData)`. The user's project shouldn't need to know the filename or set it here.